scipy/stats/_multivariate.py

```diff
@@ -786,6 +786,39 @@ def entropy(self, mean=None, cov=1):
         dim, mean, cov_object = self._process_parameters(mean, cov)
         return 0.5 * (cov_object.rank * (_LOG_2PI + 1) + cov_object.log_pdet)
 
+    def fit(self, x):
+        """Fit the multivariate normal to data `x`.
```

Comment:
```suggestion
        """Fit a multivariate normal distribution to data.
```
The main point is:
<img width="931" alt="image" src="https://github.com/scipy/scipy/assets/6570539/cb8c29b9-bb06-4c8e-a325-55f284249452">